packages/editor/src/client/panel/embedEditPanel.tsx

```diff
@@ -119,13 +126,19 @@ export function EmbedEditPanel({value, onClose, onConfirm}: EmbedEditPanel) {
         />
         <Box marginBottom={Spacing.ExtraSmall}>
           <Typography variant="subtitle1" spacing="small">
-            If you want to include a Facebook Post/Video, Instagram Post, Twitter Tweet, Vimeo
-            Video, YouTube Video you have to add a link e.g. https://www.facebook.com/id/posts/id/.
+            Facebook Post/Video, Instagram Post, Twitter Tweet, Vimeo Video, YouTube Video formatted
+            like e.g.
+          </Typography>
+          <code>https://www.facebook.com/id/posts/id/</code>
+          <Typography variant="subtitle1" spacing="small">
+            Embed codes attributes <code>title, src, width, height</code> are validated e.g.
           </Typography>
+          <code>{'<iframe width="560" height="315" src="https://..."></iframe>'}</code>
           <Typography variant="subtitle1" spacing="small">
-            In iframe embed codes the src="", width="" and height="" are validated e.g.
-            {'<iframe src="https://www.youtube.com/embed/id" width="560" height="315"></iframe>'}
+            Set <code>style</code> alternatively as attribute to overwrite the defaults and disable
+            auto ratio size scaling .e.g.
           </Typography>
+          <code>{'<iframe style="height:350px;width:100%" src="https://..."></iframe>'}</code>
```

Comment:
missing semicolon in the style attribute at the end.

Reply:
not mandatory on last item, but sure i can add it